rpc/common/moc_common_nodeinfo.proto

```diff
@@ -27,6 +31,10 @@ message GPU {
 	uint32 count = 3;
 }
 
+message OperatingSystemInfo {
+	uint64 operatingsystemsku = 1;
```

Comment:
Can we come up with enums for the operatingsystemsku, manufacturer, cputype, architecture?

Reply:
We discussed offline (along with Jocelyn, Randy),
For OperatingSystemSku/Architecture, for this PR i.e. September release, we will be just sending in the Sku number and then in Kusto we will do the mapping (i.e. maintain the list in the backend). 
Next PR: We will be adding work-items to create an Xml parser that would generate a list for us and then we can add that list in the Moc code/Kusto.
Similarly, for the other fields.

Reply:
Does that sound reasonable ask ? I can surely take it up in the next PR as an action item to add all the enums.

Operating System sku : [WindowsEditions.xml - Repos (visualstudio.com)](https://microsoft.visualstudio.com/OS/_git/os.2020?path=%2Fdata%2FWindowsEditions.xml&version=GBofficial%2Frsmaster&_a=contents)

Architecture here:
https://docs.microsoft.com/en-us/windows/win32/cimwin32prov/win32-processor#properties

Reply:
if we change the type of this property later, it would be a breaking change.
If we know the list already, I would suggest adding the enum now. The enum values can be expanded later, but closing of type of important.